sys/ztimer/Makefile.dep

```diff
@@ -8,6 +8,10 @@ ifneq (,$(filter ztimer,$(USEMODULE)))
   USEMODULE += ztimer_convert_frac
   USEMODULE += ztimer_convert_shift
 
+  # If ztimer is used enable rtt autoselection for ztimer_msec, and ztimer_sec.
+  # This will enable inclusion of `ztimer_periph_rtt` if `periph_rtt` is supported.
+  DEFAULT_MODULE += ztimer_periph_rtt_auto_select
```

Comment:
Ajj true, that is why initially I had put in Makefile.default..., couldn't remember why and moved it out :/

Reply:
I can't think of another solution than ae0bf90e7935e954908c7ea1705315c004f3dd4c... it does mean that a `ztimer` module will show up even when `ztimer` is not used...